src/Apps/Order2/Routes/Checkout/Components/FulfillmentDetailsStep/Order2FulfillmentDetailsStep.tsx

```diff
@@ -32,6 +33,11 @@ export const Order2FulfillmentDetailsStep: React.FC<
     option => option.type === "PICKUP",
   )
 
+  // biome-ignore lint/correctness/useExhaustiveDependencies: we don't want this to update
+  const initialPickupSelected = useMemo(() => {
+    return pickupOption?.selected ?? false
+  }, [])
+
```

Comment:
Fix for the UI changing as express checkout happens. I guess this is because the `fulfillmentOptions` field on the order is required in the mutation responses for express checkout, which must be getting picked up even without an explicit reference to this component's fragment.